packages/frontpage/lib/auth.ts

```diff
@@ -65,24 +65,10 @@ export const getClientMetadata = cache(async () => {
 
   const appUrl = `https://${host}`;
 
-  return {
-    // Client ID is the URL of the client metadata
-    // This isn't immediately obvious and if you supply something else the PAR request will fail with a 400 "Invalid url" error. I had to traverse the atproto implementation to find out why!
-    client_id: `${appUrl}/oauth/client-metadata.json`,
-    dpop_bound_access_tokens: true,
-    application_type: "web",
-    subject_type: "public",
-    grant_types: ["authorization_code", "refresh_token"],
-    response_types: ["code"], // TODO: "code id_token"?
-    // TODO: Tweak these?
-    scope: "atproto transition:generic",
-    client_name: "Frontpage",
-    token_endpoint_auth_method: "private_key_jwt",
-    token_endpoint_auth_signing_alg: "ES256",
-    redirect_uris: [`${appUrl}/oauth/callback`] as const,
-    client_uri: appUrl,
-    jwks_uri: `${appUrl}/oauth/jwks.json`,
-  } satisfies OAuthClientMetadata;
+  return createClientMetadata({
```

Comment:
This is the main change to watch out for but should be functionally equivalent to what we have in main right now. I'll change it to point to the new preview client in a followup PR.